### ISSUE TITLE
NuGet Gallery request returns duplicates

### ISSUE THREAD

**Issue by ghost** [NuGet/NuGetGallery#1343]
Copied from https://nuget.codeplex.com/workitem/3480

This is the request sent from VS2012: 
https://nuget.org/api/v2/Search()?$filter=IsLatestVersion&searchTerm='moq'&targetFramework='net45'&includePrerelease=false 

 NuGet.org returns 40 results (the max of a page) and says there are more at the following address: 
 <link rel="next" href="https://nuget.org/api/v2/Search?searchTerm='moq'&targetFramework='net45'&includePrerelease=false&$filter=IsLatestVersion&$skiptoken='VerificationFakes','0.1.1'" /> 

 So basically it gives a total of 42 results but VS2012 only shows 4 pages of 10 (40 results). 
 After looking at the results, there are duplicates like (Id='Xrm.Framework.Test.Moq',Version='2.0.0.1') 
 Which means that somehow VS eliminates duplications but the server should be responsible for clearing them.

![nuget_bug](https://f.cloud.github.com/assets/893557/823118/1cceb4c0-f002-11e2-9b1e-0813271a0344.PNG)
